Index: docroot/core/modules/content_moderation/src/EntityTypeInfo.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/core/modules/content_moderation/src/EntityTypeInfo.php	(revision )
+++ b/core/modules/content_moderation/src/EntityTypeInfo.php	(revision )
@@ -155,7 +155,7 @@
     }

     if (!$type->hasLinkTemplate('latest-version') && $type->hasLinkTemplate('canonical')) {
-      $type->setLinkTemplate('latest-version', $type->getLinkTemplate('canonical') . '/latest');
+      $type->setLinkTemplate('latest-version', $type->getLinkTemplate('canonical'));
     }

     // @todo Core forgot to add a direct way to manipulate route_provider, so
