Index: docroot/modules/contrib/features/src/FeaturesManager.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/docroot/modules/contrib/features/src/FeaturesManager.php	(revision )
+++ b/docroot/modules/contrib/features/src/FeaturesManager.php	(revision )
@@ -1334,6 +1334,9 @@

     // If config data is not specified, load it from the extension storage.
     foreach ($config_to_create as $name => $item) {
+      if(is_object($item)) {
+        $config_to_create[$name] = $item->getData();
+      }
       if (empty($item)) {
         $config = $this->configReverter->getFromExtension('', $name);
         // For testing purposes, if it couldn't load from a module, get config
